build.sbt

```diff
@@ -25,7 +28,34 @@ lazy val root = (project in file("."))
       "-Ywarn-dead-code",
       "-Ywarn-unused:-imports,_",
       "-unchecked"),
-    libraryDependencies ++= deps
-  )
+  libraryDependencies ++= standardDeps)
+
+//lazy val root = (project in file("."))
+//  .aggregate(serializer.projectRefs ++ codecs.projectRefs: _*)
+//  .settings(name := "akka-safer-serializer", crossScalaVersions := Nil, publish / skip := true)
+
+lazy val serializer = (projectMatrix in file("serializer"))
+  .dependsOn(codecs)
+  .settings(name := "borer-akka-serializer")
+  .settings(commonSettings)
+  .settings(libraryDependencies ++= {
+    virtualAxes.value.collectFirst({ case x: ScalaVersionAxis => x.value }).fold(Seq[ModuleID]()) {
+      case "2.13" => scala213Deps
+      case "2.12" => scala212Deps
+    }
```

Comment:
Nit:
```suggestion
    virtualAxes.value.collectFirst { case x: ScalaVersionAxis => x.value }.map {
      case "2.13" => scala213Deps
      case "2.12" => scala212Deps
    }.getOrElse(Seq.empty)```

Reply:
I find `fold` on Options hard to read, `map` + `getOrElse` is easier to reason about